Assets/Tests/Runtime/NetworkClient/NetworkClientSendIfDisconnectedTest.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using NSubstitute;
+using NUnit.Framework;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+namespace Mirage.Tests.Runtime
+{
+    public class NetworkClientSendIfDisconnectedTest
+    {
+        public GameObject clientGO;
+        public NetworkClient clientNetClient;
+
+        [SetUp]
+        public void SetUp()
+        {
+            // Initialize...
+            clientGO = new GameObject("Mirage Network Client Object", typeof(NetworkClient));
+            clientNetClient = clientGO.GetComponent<NetworkClient>();
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            Object.DestroyImmediate(clientGO);
+        }
+
+        [Test]
+        // Ensure that the created client game object and it's component(s) isn't null.
+        public void EnsureNetworkClientIsNotNull()
+        {
+            Assert.IsNotNull(clientGO);
+            Assert.IsNotNull(clientNetClient);
+        }
+
+        [Test]
+        // Ensure that the client object's Player reference is null.
+        public void EnsureNetworkClientPlayerIsNull()
+        {
+            Assert.IsNull(clientNetClient.Player);
+        }
```

Comment:
we probably dont need these tests as they are just validating the setup, 

I think debug.assert is better for stuff like this because these checks are more for debugging a test if it goes wrong

Reply:
Alright, makes sense… I’ll use Debug.Assert in the setup routine.